docs/Server.md

```diff
@@ -224,6 +224,32 @@ const fastify = require('fastify')({
 })
 ```
 
+<a name="versioning"></a>
+### `versioning`
+
+By default you can version your routes with [semver versioning](https://github.com/fastify/fastify/blob/master/docs/Routes.md#version), which is provided by `find-my-way`. There is still an option to provide custom versioning strategy. You can find more information in the [find-my-way](https://github.com/delvedor/find-my-way#versioned-routes) documentation.
```

Comment:
will this be converted to the correct link to the documentation when `.md` files will be built? IDK, but saw this in other part of the docs.

Reply:
cc @lmammino

Reply:
I think it will, here in the docs: https://www.fastify.io/docs/latest/Server/#server it points to the documentation. However I have no idea how to check this :)